.gitignore

```diff
@@ -21,3 +21,9 @@ config.local
 # https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
 /.env.*.local
 /.env.local
+
+# Intellij files
+.idea/
+
+# Ruby version
+.ruby-version
```

Comment:
This is fine - we don't need .ruby-version since bundler and rvm can both pluck it from `Gemfile` instead, and one source of truth for the Ruby version is better than two :-)